internal/analysis/realtime.go

```diff
@@ -44,6 +47,20 @@ func RealtimeAnalysis(settings *conf.Settings) error {
 	// Initialize database access.
 	dataStore := datastore.New(settings)
 
+	// Initialize Prometheus (if enabled)
+	if settings.Realtime.Prometheus {
+		settings.Realtime.PrometheusDetectionCounter = prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "birdnet_detections",
```

Comment:
As this is a counter it probably should have ended with `_total` or `_count`. You'll probably see warnings like this when using PromQL.

> New warning in edge: PromQL info: metric might not be a counter, name does not end in _total/_sum/_count/_bucket

Reply:
Hey nice catch! I don't know anything about PromQL but I see no reason why this small change shouldn't be applied. 

I think my (selfish) goal of this PR was simply to introduce an option to use existing visualization systems instead of building yet another HTML dashboard ( not saying there's anything wrong with the existing one <3 ).

Feel free to make this change if @tphakala's on board.